readme.md

```diff
@@ -17,24 +17,33 @@ $ npm install --save skin-tone
 ```js
 const skinTone = require('skin-tone');
 
-skinTone('👍', 4);
+skinTone('👍', skinTone.BROWN); // or skinTone('👍', 4);
 //=> '👍🏾'
 
-skinTone('👍', 1);
+skinTone('👍', skinTone.WHITE); // or skinTone('👍', 1);
 //=> '👍🏻'
 
 // can also remove skin tone
-skinTone('👍🏾', 0);
+skinTone('👍🏾', skinTone.NONE); // or skinTone('👍🏾', 0);
 //=> '👍'
 
 // just passes it through when not supported
-skinTone('🦄', 5);
+skinTone('🦄', skinTone.DARK_BROWN); // or skinTone('🦄', 5);
```

Comment:
Wanted to do it like this but I was afraid it would be too confusing, if someone were to understand it as both lines are required. Therefore I wanted to make it as clear as possible that one or the other can be used.